go/enclave/gas/gas.go

```diff
@@ -4,12 +4,12 @@ import (
 	"math/big"
 )
 
-// we choose a very conservative compression factor of 10%
+// we choose a very conservative compression factor
 // in practice, for most transactions, it will be much better
-const compressionFactor = 90
+const compressionFactor = 60
```

Comment:
we should probably make this configurable in the future through some system smart contract

Reply:
I saw others has it hardcoded at like 90%. I think it's a bit overkill to configure this. We can tweak between versions